templates/elements/quote/quote.html.twig

```diff
@@ -1,6 +1,6 @@
 <div class="decanter-quote">
-  <img class="decanter-quote__img " src="{{ image }}" alt="{{ altText }}">
-  <div class="decanter-quote__body ">
+  <img class="decanter-quote__img" src="{{ imageSource }}" alt="{{ alt }}">
```

Comment:
@josephgknox 

Do not use camelCase for variable names

The standard is to use underscore. See: https://twig.symfony.com/doc/2.x/coding_standards.html

change imageSource to `image_source` or better yet the shorter `img_source` or even shorter `src`

Reply:
For sure! Thanks! Fixed.